### DISCUSSION TITLE
Slug BUG

### DISCUSSION THREAD

**Discussion by jozoXXL** [trendyminds/craft-palette#24] [Q&A] [answered]
There is a bug with HIGH ASCII letters in slug.. 

when slug is like this..  all works OK::   Edit entry is shown
### **exclusive-interview-with-dana-white**

![Snag_3934786](https://user-images.githubusercontent.com/7314171/212013569-6fd9bbc3-7d5d-4ebe-bd08-c03aba9f732b.png)

When there is  HIGH ASCII in slug .. not working 
### exclusive-interview-with-dana-white-**šđčćž**
![Snag_393d29f](https://user-images.githubusercontent.com/7314171/212013657-aa2147fc-d481-4eb7-9431-aec42de3d0ec.png)

**Comment by wmdhosting** [comment]
i installed todays update.. 
But still edit link not visible..with HIGH ASCII letters..

**Comment by wmdhosting** [comment]
Can you fix this one. i will pay for it.. no problem

**Comment by aaronbushnell** [comment, marked as answer]
This is fixed as of 3.1.3 #30

**Comment by wmdhosting** [comment]
Thanks for  response.  i thinked too that was fixed.. 

but its not..  i just tested now..  and not fixed..

**Reply by aaronbushnell** [reply]
This works okay for me—here's a slug with special characters:
![Screenshot 2023-03-31 at 1 19 16 PM](https://user-images.githubusercontent.com/315202/229187554-1142cc25-366b-4f4c-8b34-09a1ca956e7b.png)

**Comment by wmdhosting** [comment]
OK thx for info. 
Will check why  not work on my side..
